app/src/main/java/com/eventyay/organizer/core/attendee/history/CheckInHistoryView.java

```diff
@@ -6,5 +6,5 @@
 import com.eventyay.organizer.common.mvp.view.Refreshable;
 import com.eventyay.organizer.data.attendee.CheckInDetail;
 
-public interface CheckInHistoryView extends Progressive, Erroneous, Refreshable, Emptiable<CheckInDetail> {
```

Comment:
Don't remove this. Just leave the onRefreshComplete method empty with a comment

Reply:
Okay